src/http/router.ts

```diff
@@ -677,8 +677,8 @@ export const getRouter = (client: CLClient, options: ICLClientOptions): express.
         try {
             const key = getMemoryKey(req)
             // Update Memory
-            const memory = CLMemory.GetMemory(key)
-            await memory.BotMemory.ClearAsync();
+            const memory = CLStorage.Get(key)
```

Comment:
might make these 'const storage ='

Reply:
Yea, I will do.

I had decided again these usage of storage renames because I had thought after changing the hierarchy we wouldn't directly expose storage here, but some other object composed of the three states thus those renaming changes would be obsolete.